src/status_im2/common/contacts/views.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im2.common.contacts.views
+  (:require [status-im2.common.contacts.style :as style]
+            [i18n.i18n :as i18n]
+            [utils.debounce :as debounce]
+            [utils.re-frame :as rf]
+            [react-native.core :as rn]
+            [quo2.core :as quo2]
+            [quo2.foundations.colors :as colors]))
+
+(defn new-contact
+  []
+  (let [{:keys [public-key state error]}
+        (rf/sub [:contacts/new-identity])]
+    [rn/view
+     [rn/image {:source (js/require "../resources/images/ui/add-contact.png")}]
```

Comment:
I think the `js/require` could be moved to a private constant to improve readability

Reply:
yes, thank you